AEPIdentity/Tests/IdentityTests.swift

```diff
@@ -485,6 +485,19 @@ class IdentityTests: XCTestCase {
         XCTAssertTrue(identity.readyForEvent(firstEvent))
     }
 
+    func testReadyForEventBootupWithValidLastSetConfigurationReturnsTrue() {
```

Comment:
Can we also add a test to IdentityFunctionalTests.swift that checks that a sync event uses last config set?
setup:
simulate config shared state 1 orgid privacy optunknown, .set
simulate config shared state 2 orgid privacy optin, .set
simulate config shared state 3 nil, .pending
send syncIdentifier event
assert last valid config 2 used, and network call sent with orgid